_test/export.response.test.php

```diff
@@ -0,0 +1,191 @@
+<?php
+/**
+ * General tests for the findologicxmlexport plugin
+ *
+ * @group plugin_findologicxmlexport
+ * @group plugins
+ */
+
+//======================================================================
+// EXPORT RESPONSE TESTS
+//======================================================================
+
+require_once(__DIR__ . '/../DokuwikiXMLExport.php');
+require_once(__DIR__ . '/../admin.php');
+require_once(__DIR__ . '/helper.php');
+
+class export_response_test extends DokuWikiTest
+{
+    public function setUp()
+    {
+        $helper = new helper;
+        $helper->setUp();
+    }
+
+    /**
+     * @dataProvider parameterProviderForXMLResponse
+     */
+    public function test_parameters_start_count_and_total_are_valid_for_($ids = array())
+    {
+        $helper = new helper;
+        if ($ids) { // I also want to check for an empty DokuWiki
+            $helper->savePages(array($ids));
+            $expectedCount = count($ids);
```

Comment:
Maybe these should be provided by the data provider, then you don't need to distinguish in the test code.